### DISCUSSION TITLE
Installing as an extension in Text Gen Webui

### DISCUSSION THREAD

**Discussion by Bobbyjohnsonz** [openai/whisper#2046] [Q&A]
Tried on Linux Ubuntu and Mint same issue. Whisper is working in windows just fine with the exact same setup. 

After installing sudo apt install ffmpeg, Whisper runs without any error however it just processes forever and never completes. 

I've tried reinstalling however there is a strange dependency error with text gen webui. This is whisper stt, I'm not sure if there's a difference but I've also tried with the clone here and got the exact same error.

pip install --upgrade -r requirements.txt

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
torchvision 0.16.2+cu121 requires torch==2.1.2, but you have torch 2.2.1 which is incompatible.
torchaudio 2.1.2+cu121 requires torch==2.1.2, but you have torch 2.2.1 which is incompatible.
datasets 2.17.1 requires fsspec[http]<=2023.10.0,>=2023.1.0, but you have fsspec 2024.2.0 which is incompatible.

**Comment by phineas-pta** [comment]
try update ffmpeg to latest v6

default apt installation not always give latest version, something like v4
